### PR TITLE
Title case tooltips

### LINKED ISSUE #2266: Make tooltips title case; also no periods
Per Eclipse Guideline 1.5: https://wiki.eclipse.org/User_Interface_Guidelines#Language

"Use Headline style capitalization for menus, tooltip and all titles, including those used for windows, dialogs, tabs, column headings and push buttons. Capitalize the first and last words, and all nouns, pronouns, adjectives, verbs and adverbs. Do not include ending punctuation."

### PR DESCRIPTION
**Pull request by elharo**
@chanseokoh fix #2266 per Eclipse UI guidelines

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties [hunk 1]
```diff
@@ -43,20 +43,20 @@ appengine.java.component.missing=Could not deploy project because the App Engine
...
-tooltip.manual.promote.link=To manually promote a version use the Google Cloud Console: {0}
-tooltip.stop.previous.version=If checked, stops the previously running version when deploying a \
- new version that receives all traffic.
-tooltip.staging.bucket=The Google Cloud Storage bucket used to stage files for deployment. \
- If not specified, the application''s default code bucket is used.
+tooltip.account=Google Account Used to Deploy the App to the Google Cloud Platform
+tooltip.project.id=The Google Cloud Platform Project where the App Will Be Deployed
+tooltip.app.yaml=app.yaml Path, either Absolute or Relative to the Project
+tooltip.version=The Version of the App that Will Be Created or Replaced by this \
+ Deployment. If You Do Not Specify a Version, One Will Be Generated for You.
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties]: No ending period.

File: plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties [hunk 2]
```diff
@@ -43,20 +43,20 @@ appengine.java.component.missing=Could not deploy project because the App Engine
...
+tooltip.manual.promote.link=To Manually Promote a Version Use the Google Cloud Console: {0}
+tooltip.stop.previous.version=If Checked, Stops the Previously Running Version when Deploying a \
+ New Version that Receives All Traffic
+tooltip.staging.bucket=The Google Cloud Storage Bucket Used to Stage Files for Deployment. \
+ If Not Specified, the Application''s Default Code Bucket Is Used.
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties]: No ending period.

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties [hunk 3]
```diff
@@ -28,8 +28,8 @@ MAVEN_PROJECT_COORDINATES=Maven project coordinates
 GROUP_ID=Group ID:
 ARTIFACT_ID=Artifact ID:
 ARTIFACT_VERSION=Version:
-GROUP_ID_TOOLTIP=The Maven Group ID. Should be an alphanumeric path separated by periods.
-ARTIFACT_ID_TOOLTIP=The Maven Artifact ID. Should be an alphanumeric name separated by dashes.
+GROUP_ID_TOOLTIP=The Maven Group ID. Should Be an Alphanumeric Path Separated by Periods.
+ARTIFACT_ID_TOOLTIP=The Maven Artifact ID. Should Be an Alphanumeric Name Separated by Dashes.
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties]: No ending period for these two.

File: plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui.test/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/flexible/FlexDeployPreferencesPanelTest.java [hunk 4]
```diff
@@ -132,11 +130,19 @@ private static IStatus getAppYamlPathValidationStatus(FlexDeployPreferencesPanel
...
       public boolean apply(Control control) {
-        return control instanceof Text && APP_YAML_FIELD_TOOLTIP.equals(control.getToolTipText());
+        return control instanceof Text && 
+            "app.yaml Path, either Absolute or Relative to the Project".equals(
+                control.getToolTipText());
       }
-    });
+    };
+    
+    Control control = CompositeUtil.findControl(panel, predicate);
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui.test/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/flexible/FlexDeployPreferencesPanelTest.java]: I expect this whole thing can be simplified with `findControlAfterLabel()` which was introduce after this test was written.

```java
Control control = CompositeUtil.findControlAfterLabel(panel, Text.class, "app.yaml:");
```

Want to try?

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties (hunk 3 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties]: done

File: plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties (hunk 1 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties]: done

File: plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties (hunk 2 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties]: done

**Pull request merged**